main.go

```diff
@@ -63,7 +63,22 @@ func tryUnpin(path string, sh *shell.Shell) error {
 	return nil
 }
 
-func Pin(b *hb.Bot, actor, path string) {
+var pinfile = "pins.log"
+
+func writePin(pin, label string) error {
+	fi, err := os.OpenFile(pinfile, os.O_APPEND|os.O_EXCL|os.O_WRONLY, 0660)
+	if err != nil {
+		return err
+	}
+
+	_, err = fmt.Fprintf(fi, "%s\t%s\n", pin, label)
```

Comment:
can irc messages contain whitespace? if they do, we should escape it, so that we don't need to take care of it when parsing the log